coderd/files_test.go

```diff
@@ -101,14 +106,7 @@ func TestDownload(t *testing.T) {
 		_ = coderdtest.CreateFirstUser(t, client)
 
 		// given
-		tarball, err := echo.Tar(&echo.Responses{
-			Parse:          echo.ParseComplete,
-			ProvisionApply: echo.ApplyComplete,
-		})
-		require.NoError(t, err)
-
-		tarReader := tar.NewReader(bytes.NewReader(tarball))
-		zipContent, err := coderd.CreateZipFromTar(tarReader)
+		zipContent, err := os.ReadFile(filepath.Join(".", "testdata", "test.zip"))
```

Comment:
Nit: Simplify, it's already relative without the `.`.

```suggestion
		zipContent, err := os.ReadFile(filepath.Join("testdata", "test.zip"))
```